docs/authentication/subscription-authentication.md

```diff
@@ -0,0 +1,37 @@
+# Authenticating subscriptions
+Some users require authentication and authorization to be able to communicate with their APIs. For this we will support Maskinporten and Entra Id from Microsoft.
+## Entra Id
+Entra Id is the standard OAuth authentication mechanism used in Azure. To be able to use this with Event Issuer subscriptions, we will need to do a server-to-server interaction that runs in the background, without immediate interaction with a user. This is done through an OAuth client-credential flow that grants permissions directly to the application itself by an administrator.
+
+### Client credentials flow 
+This diagram describes how authentication works between background services.
+
+For Event-Issuer the flow would look like this
+``` mermaid
```

Comment:
As the Admin app registration creation is not relevant in this case, can be included as a pre-condition instead in the diagram itself.